typings/index.d.ts

```diff
@@ -2,7 +2,7 @@ declare module '@paralleldrive/react-feature-toggles' {
   import * as React from 'react';
 
   type Feature1 = {
```

Comment:
Yes, that looks correct now. I don't see a reason for these Types declarations, it's actually kind of confusing. Can we remove them and make them part of the function declarations?

Also, the second function declaration doesn't seem to give a return type.

```ts
function Feature({
  children
} : {
  children: ({ features: ReadonlyArray<string> }) => React.ReactNode
}): React.ReactNode;

function Feature({
  activeComponent
  inactiveComponent,
  name,
 } : {
  activeComponent: () =>  React.ReactNode,
  inactiveComponent: () => React.ReactNode,
  name: string
}): React.ReactNode
```